Source/WebCore/crypto/mac/CryptoKeyECMac.cpp

```diff
@@ -143,8 +195,8 @@ RefPtr<CryptoKeyEC> CryptoKeyEC::platformImportJWKPublic(CryptoAlgorithmIdentifi
     CCECCryptorRef ccPublicKey = nullptr;
     if (CCECCryptorCreateFromData(size, x.data(), x.size(), y.data(), y.size(), &ccPublicKey))
         return nullptr;
-
-    return create(identifier, curve, CryptoKeyType::Public, PlatformECKeyContainer(ccPublicKey), extractable, usages);
+    
```

Comment:
Bad spacing change. Please change your IDE settings if it is your IDE doing this.

Reply:
changed